CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/visual/DefaultInteractionEventHandler.java

```diff
@@ -972,26 +971,18 @@ private void createTransaction(final GraphWriteMethods wg, final int fromVertex,
      *
      * @return array of node IDs
      */
-    private List<Integer> gatherSelectedNodes(GraphReadMethods rg) {
-        List<Integer> selectedIds = new ArrayList<>();
-        final int vertexSelctedAttribute = VisualConcept.VertexAttribute.SELECTED.get(rg);
-        if (vertexSelctedAttribute != Graph.NOT_FOUND) {
-            rg.vertexStream().forEach(vertexId -> {
-                if (rg.getBooleanValue(vertexSelctedAttribute, vertexId)) {
-                    selectedIds.add(vertexId);
-                }
-            });
-        }
-
+    public List<Integer> gatherSelectedNodes(GraphWriteMethods wg) {
```

Comment:
Can this go back to a `GraphReadMethods`?

Reply:
Yes, done